projects/batfish/src/test/java/org/batfish/main/BatfishTest.java

```diff
@@ -176,6 +178,90 @@ public void testParseTopologyJson() throws IOException {
     assertEquals(topology.getEdges().size(), 2);
   }
 
+  //positive example
+  @Test
+  public void testCheckTopologyNodePos() throws IOException {
+    String topologyJson =
+        "["
+            + "{ "
+            + "\"node1\" : \"h1\","
+            + "\"node1interface\" : \"eth0\","
+            + "\"node2\" : \"h2\","
+            + "\"node2interface\" : \"e0\""
+            + "}"
+            + "]";
+    Path topologyFilePath =
+        CommonUtil.createTempFileWithContent("testParseTopologyJson", topologyJson);
```

Comment:
Can you simplify this setup code, perhaps with some `private static` utility methods. Things that come to mind:

For building a topology, it's simpler & more readable to build a Java map yourself and then let Jackson do the work of serialization.

```java
List<Map<String, String>> topology = Collections.singletonList(
    ImmutableMap.<String, String>builder()
         .add("n1", "i1")
         .add...
         .build());
//use BatfishObjectMapper to serialize it to a file.
```
and that probably gets wrapped in a utility method.

Making a new host Configuration for a given host/ethernet pair definitely should go in a utility - you do it 5 times.